<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>600</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Deplieur UI</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QVBoxLayout" name="verticalLayout">
    <item>
     <widget class="QSlider" name="hSlider">
      <property name="enabled">
       <bool>false</bool>
      </property>
      <property name="minimum">
       <number>-180</number>
      </property>
      <property name="maximum">
       <number>180</number>
      </property>
      <property name="orientation">
       <enum>Qt::Horizontal</enum>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QGraphicsView" name="graphicsView">
      <property name="autoFillBackground">
       <bool>false</bool>
      </property>
      <property name="verticalScrollBarPolicy">
       <enum>Qt::ScrollBarAlwaysOn</enum>
      </property>
      <property name="horizontalScrollBarPolicy">
       <enum>Qt::ScrollBarAlwaysOn</enum>
      </property>
      <property name="sizeAdjustPolicy">
       <enum>QAbstractScrollArea::AdjustToContents</enum>
      </property>
      <property name="backgroundBrush">
       <brush brushstyle="NoBrush">
        <color alpha="255">
         <red>0</red>
         <green>0</green>
         <blue>0</blue>
        </color>
       </brush>
      </property>
      <property name="sceneRect">
       <rectf>
        <x>0.000000000000000</x>
        <y>0.000000000000000</y>
        <width>1500.000000000000000</width>
        <height>900.000000000000000</height>
       </rectf>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>800</width>
     <height>22</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuFichier">
    <property name="title">
     <string>Fichier</string>
    </property>
    <addaction name="separator"/>
    <addaction name="actionNouveau"/>
    <addaction name="actionOuvrir"/>
    <addaction name="actionSauver"/>
    <addaction name="separator"/>
    <addaction name="actionExporter"/>
    <addaction name="separator"/>
    <addaction name="actionQuitter"/>
   </widget>
   <widget class="QMenu" name="menuAffichage">
    <property name="title">
     <string>Affichage</string>
    </property>
    <addaction name="actionZoom_Plus"/>
    <addaction name="actionZoom_Normal"/>
    <addaction name="actionZoom_Moins"/>
   </widget>
   <widget class="QMenu" name="menuPiece">
    <property name="title">
     <string>Piece</string>
    </property>
    <addaction name="actionTourner_15_Droite"/>
    <addaction name="actionTourner_5_Droite"/>
    <addaction name="actionTourner_5_Gauche"/>
    <addaction name="actionTourner_15_Gauche"/>
    <addaction name="separator"/>
    <addaction name="actionMonter"/>
    <addaction name="actionDroite"/>
    <addaction name="actionGauche"/>
    <addaction name="actionDescendre"/>
   </widget>
   <addaction name="menuFichier"/>
   <addaction name="menuAffichage"/>
   <addaction name="menuPiece"/>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
  <action name="actionNouveau">
   <property name="text">
    <string>Nouveau</string>
   </property>
  </action>
  <action name="actionOuvrir">
   <property name="text">
    <string>Ouvrir</string>
   </property>
  </action>
  <action name="actionExporter">
   <property name="text">
    <string>Exporter</string>
   </property>
  </action>
  <action name="actionQuitter">
   <property name="text">
    <string>Quitter</string>
   </property>
  </action>
  <action name="actionZoom">
   <property name="text">
    <string>Zoom +</string>
   </property>
  </action>
  <action name="actionZoom_Plus">
   <property name="text">
    <string>Zoom +</string>
   </property>
   <property name="shortcut">
    <string>Ctrl++</string>
   </property>
  </action>
  <action name="actionZoom_Normal">
   <property name="text">
    <string>Zoom 1:1</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+0</string>
   </property>
  </action>
  <action name="actionZoom_Moins">
   <property name="text">
    <string>Zoom -</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+-</string>
   </property>
  </action>
  <action name="actionSauver">
   <property name="text">
    <string>Sauver</string>
   </property>
  </action>
  <action name="actionTourner_15_Droite">
   <property name="text">
    <string>Tourner 15° Droite</string>
   </property>
  </action>
  <action name="actionTourner_5_Droite">
   <property name="text">
    <string>Tourner 5° Droite</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+PgDown</string>
   </property>
   <property name="shortcutContext">
    <enum>Qt::ApplicationShortcut</enum>
   </property>
  </action>
  <action name="actionTourner_5_Gauche">
   <property name="text">
    <string>Tourner 5° Gauche</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+PgUp</string>
   </property>
   <property name="shortcutContext">
    <enum>Qt::ApplicationShortcut</enum>
   </property>
  </action>
  <action name="actionTourner_14_Gauehc">
   <property name="text">
    <string>Tourner 14° Gauehc</string>
   </property>
  </action>
  <action name="actionTourner_15_Gauche">
   <property name="text">
    <string>Tourner 15° Gauche</string>
   </property>
  </action>
  <action name="actionMonter">
   <property name="text">
    <string>Monter</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+Up</string>
   </property>
   <property name="shortcutContext">
    <enum>Qt::ApplicationShortcut</enum>
   </property>
  </action>
  <action name="actionDroite">
   <property name="text">
    <string>Droite</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+Right</string>
   </property>
   <property name="shortcutContext">
    <enum>Qt::ApplicationShortcut</enum>
   </property>
  </action>
  <action name="actionGauche">
   <property name="text">
    <string>Gauche</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+Left</string>
   </property>
   <property name="shortcutContext">
    <enum>Qt::ApplicationShortcut</enum>
   </property>
  </action>
  <action name="actionDescendre">
   <property name="text">
    <string>Descendre</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+Down</string>
   </property>
   <property name="shortcutContext">
    <enum>Qt::ApplicationShortcut</enum>
   </property>
  </action>
 </widget>
 <resources/>
 <connections/>
</ui>
